File: Find Minimum/FindMin.java
public class FindMin {
    public static int min(int[] arr) {
        int target = arr[0];
        int index = 0;

        for (int i : arr) {
            if (i < target) {
                target = arr[index];
            }
            index++;
        }
        return target;
    }

    public static void main(String[] args) {
        int[] array = {13, 44, 576, 87, -12, 0, -45, 78};
        System.out.println(min(array));
    }
}

File: Find Minimum/findMin.ts
function findMin(arr: number[]) {
    let min = arr[0];
    for (let i = 1; i <= arr.length-1; i++) {
        if (arr[i] < min) {
            min = arr[i];
        }
    }
    return min;
}

let arr: number[] = [13, 44, 576, 87, -12, 0, -45, 78];

console.log(findMin(arr))
